people/anhelina.json

```diff
@@ -0,0 +1,21 @@
+{
+
+  "key": "anhelina",
+
+  "name": "Anhelina Sadovnikova",
+
+  "nick": "Anhelina",
+
+  "email": "anhelina.sadovnikova@mild.blue",
+
+  "title_cs": "FRONTEND VÝVOJÁŘKA",
+
+  "title_en": "FRONTEND DEVELOPER",
+
+  "short_desc_en": "Anhelina is a student of the Faculty of Biomedical Engineering of the Czech Technical University in Prague, majoring in Informatics and Cybernetics in Healthcare. Works at Mild Blue as a frontend developer.",
+
+  "short_desc_de": "Anhelina ist Studentin der Fakultät für Biomedizinische Technik der Tschechischen Technischen Universität. Arbeitet bei Mild Blue als Frontend-Entwickler.",
+
+  "short_desc_cs": "Anhelina je studentkou Fakulty biomedicínského inženýrství ČVUT, oboru Informatika a kybernetika ve zdravotnictví. Pracuje v Mild Blue jako frontend vývojářka."
```

Comment:
mohla bys to take trochu rozvest. Treba napsat odkud jsi, co te zajima, co delas, ceho chces v Mild Blue (nebo v zivote) dosahnout atd. At je to trochu osobni, to pusobi dobre